engine/runtime/src/main/java/org/enso/interpreter/node/callable/dispatch/LoopingCallOptimiserNode.java

```diff
@@ -79,13 +82,22 @@ public Object uncachedDispatch(
       State state,
       Object[] arguments,
       @Cached ExecuteCallNode executeCallNode) {
+    Warning[] warnings = null;
     while (true) {
       try {
-        return executeCallNode.executeCall(frame, function, callerInfo, state, arguments);
+        Object result = executeCallNode.executeCall(frame, function, callerInfo, state, arguments);
+        if (warnings != null) {
```

Comment:
I am fine that usage of `Warnings` slows execution down, however in this case we are slowing _everything_ down - be there warnings or not, this change has to be made. I'd like us to find another solution.